docs/source/ui/pages/run_management.md

```diff
@@ -33,150 +40,153 @@ Step 1: Select runs on the runs table:
+You can track the gradients, weights, and biases distributions of all layers for multiple steps using Aim.
 
-- navigate between the layers
-- search for distribution on specific steps
+The "Distributions" tab allows you to observe them for a single run and also:
+- navigate between layers 
+- search for distributions on specific steps.
 
-The single run distributions tab is quite powerful!
+The single run "Distributions" tab is quite powerful!
 
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Single Run page distributions tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/distributions_tab.png">
 
 ### Images
 
-The Images tab contains all the tracked images of a single run.
+The `Images` tab contains all the tracked images of a single run. 
 You can track runs with different contexts and at different steps of training.
 
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Single Run page images tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/images_tab.png">
 
-On the left-hand side are the names of different image-sets you have tracked along with their [context](../../understanding/concepts.html#sequence-context) unpacked.
+On the left-hand side, you will see the names of the different image sets you have tracked along with their [context](../../understanding/concepts.html#sequence-context) unpacked.
 
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Select Context" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/select_context.png">
 
-Usually the images are tracked at diff steps and with batches. This control will allow you to quickly slice and dice the specific subset of images to view.
-Use these sliders to search
+Usually, images are tracked at different steps and with batches. 
+The provided controls will allow you to quickly slice and view a specific subset of images.
+
+Use these sliders to search:
 
 - which subset of steps you'd like to see (on the left-hand side)
 - which indices you'd like to see (on the right-hand side)
 
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Slider" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/images-explore/images-explore-range-panel.png">
 
-If there is only 1 step or only 1 index, you will see the info message instead of the control
+If there is only one step or only one index, you will see an informational message instead of the control.
 
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Slider" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/range_panel_with_info_massages.png">
 
 ### Audios
 
-You can track audios with Aim. Use the Audios tab to view and play the tracked audios of a single run.
+Aim allows you to track audios. Use the `Audios` tab to view and play the tracked audios of a single run.
```

Comment:
An inconsistency - I saw other lines where the "Audio" format would be used. why are we getting rid of the `Audio` ?

Reply:
I will change here the code block to double quotes, for the same reason i already mentioned above